src/pages/DatasetSearch.jsx

```diff
@@ -411,6 +411,60 @@ export const DatasetSearch = (props) => {
       icon: gp2Icon,
       title: 'Global Parkinson\'s Genetics Program Data Library',
     },
+    'broadasd': {
+      query: {
+        match_phrase: {
+          'study.description': 'ASD',
+        },
+      },
+      icon: broadIcon,
+      title: 'Broad Institute Autism Spectrum Disorder (ASD) Data Library',
```

Comment:
```suggestion
      title: 'Broad Institute Autism Spectrum Disorder (ASD) Data Library',
```

Is the title necessary if there's an icon and url?